tests/validation/tests/v3_api/test_airgap_ha.py

```diff
@@ -159,19 +179,27 @@ def get_registry_resources(external_node):
     run_command(get_resources_command, log_out=False)
 
 
-def install_rke_helm_kubectl(external_node):
+def install_rke(external_node):
     docker_compose_command = \
         'cd ~/ && sudo wget ' \
         'https://github.com/rancher/rke/releases/download/{}' \
         '/rke_linux-amd64 && ' \
         'sudo mv rke_linux-amd64 rke && ' \
         'sudo chmod +x rke && ' \
-        'sudo mv rke /usr/local/bin && ' \
-        'sudo snap install helm --classic && ' \
-        'sudo snap install kubectl --classic'.format(RKE_VERSION)
+        'sudo mv rke /usr/local/bin'.format(RKE_VERSION)
     return external_node.execute_command(docker_compose_command)
 
 
+def install_helm(external_node):
+    return external_node.execute_command(
+        'sudo snap install helm --classic')
```

Comment:
snaps are ubuntu only iirc, are we planning to support other OS's in the future? If that's the case I think downloading the binaries would be preferable. 

Otherwise if the plan is to support Ubuntu only this is ok.

Reply:
I think Airgap for now is on ubuntu only if I am not mistaken. HA deploy (non airgap) takes in AMI and user.

Reply:
yes for now we have it for ubuntu only, but that is a good point for future requests.